build.gradle

```diff
@@ -109,17 +108,17 @@ subprojects {
   // ------------
 
   ext {
-    grpcVersion = grpcVersion
     opencensusVersion = opencensusVersion
     commonProtosVersion = commonProtosVersion
 
     // Shortcuts for libraries we are using
     libraries = [
-        grpcStub: "io.grpc:grpc-stub:${grpcVersion}",
-        grpcAuth: "io.grpc:grpc-auth:${grpcVersion}",
-        grpcProtobuf: "io.grpc:grpc-protobuf:${grpcVersion}",
-        grpcNetty: "io.grpc:grpc-netty-shaded:${grpcVersion}",
-        grpcAlts: "io.grpc:grpc-alts:${grpcVersion}",
+        grpcBom: "io.grpc:grpc-bom:1.19.0",
```

Comment:
The grpc bom is a different version from the jar itself?

Reply:
Sorry, I should have included the version update in the title (which I have just updated now) - this also bumps grpc to 1.19.0 in the process (grpc-bom isn't available for 1.18.0)